components/suggestion/emoticon_provider.tsx

```diff
@@ -148,7 +158,7 @@ export default class EmoticonProvider extends Provider {
             }
         }
 
-        const sortEmojisHelper = (a, b) => {
+        const sortEmojisHelper = (a: Emoji, b: Emoji) => {
```

Comment:
as mentioned above it would not be `Emoji`

Reply:
but the function ***sortEmojisHelper*** is returning ***compareEmojis*** function which also takes a & b argument both of type Emoji

Reply:
Yes that's correct that function expects it to be Emoji parameter but if you console log the parameters you will be that they are not of that type

Reply:
Also the function returns the result of internal function